.travis.yml

```diff
@@ -30,8 +30,13 @@ before_install:
       pip install psycopg2
     fi
 install:
-  - pip install -U pip
+  - pip install --upgrade pip
   - pip install --pre -r dev-requirements.txt .
+  - |
+    # pin tornado
+    if [[ ! -z "$TORNADO" ]]; then
+      pip install --pre tornado==$TORNADO
```

Comment:
so `pip install --pre tornado==` works for the cases where we don't set `$TORNADO`? Every day is a school day!

Reply:
Due to the `-z` check above, this install doesn't run if $TORNADO is undefined.

Reply:
Thanks, apparently I had my eyes closed when reading this.